### PR TITLE
Calibration docs

### PR DESCRIPTION
**Pull request by Edoardo-Pedicillo**
In this PR, @MatteoRobbiati and I will write the calibration routines documentation and modify the qibocal website.

TO DO:
- [x] Add the calibration routines documentation 

Checklist:
- [x] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Review by MatteoRobbiati — COMMENTED** [review]
Thank you for this @Edoardo-Pedicillo . As previously said by @andrea-pasquale, it would be great to change the fit image (to show a better fit in the doc).

**Comment by Edoardo-Pedicillo** [comment]
Thank @MatteoRobbiati, I changed the image.

**Comment by Edoardo-Pedicillo** [comment]
@andrea-pasquale, In this PR there are a lot of changes, if you agree, we can merge this branch and write the documentation of the calibration routines in a new one.

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @Edoardo-Pedicillo.
You can find below some comments.
I would suggest to modify also the image`doc/source/getting-started/qq-live_screenshot.png` since the output shows that the qubit is not calibrated.
For example you can put a plot from this  [report](http://login.qrccluster.com:9000/DI8NosXRRGutAN8CkhSkFw==/), which are quite nice, since you can clearly see that we are able to discriminate the ground and the excited states.

**Review by MatteoRobbiati — COMMENTED** [review]
Thank you for this modifications @Edoardo-Pedicillo .

**Comment by Edoardo-Pedicillo** [comment]
Tank @andrea-pasquale, let me know if it is all ok.

**Review by MatteoRobbiati — COMMENTED** [review]
Thank you @Edoardo-Pedicillo for this. 
Here some possible changes.

I'm going to open a PR because I've found some typo reading the docs. I created a new brach called `fix-typos` that can be used for open super fast PRs in order to fix these little problems.

**Review by MatteoRobbiati — COMMENTED** [review]
I was reading one time more the doc. 
I am not sure that it is useful to plot all the web page report as for example we do [here](file:///home/matteo/Documents/PhD/qibocal/doc/build/html/getting-started/interface.html). 

I'm speaking in terms of readability.
Perhaps it would be better to give a brief description of the content of the page and report the notable images (scatterplots in the case of `states_calibration`) and, if we still want to give an idea of what the page looks like when loaded properly, we could redirect with a link to these larger and more complete images.

What do you think about it @Edoardo-Pedicillo and @andrea-pasquale ?

File: doc/source/getting-started/example.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+.. code-block::
+
+    qq example.yml -o resonator_spectroscopy_routine
+
+
+Visualize the data
+^^^^^^^^^^^^^^^^^^
+
+Qibocal gives the possibility to live-plotting with the ``qq-live`` command. Opening the output link in the browser, the web page will look like the following image.
```
[the remark this answers is not in the document]
Comment (Edoardo-Pedicillo) [reply doc/source/getting-started/example.rst]: Now `qq-live` is a hypertext

File: doc/source/getting-started/installation.rst [hunk 2]
```diff
@@ -25,12 +25,6 @@ distributed with pypi* for the packages listed above.
 Qibocal
```
[the remark this answers is not in the document]
Comment (Edoardo-Pedicillo) [reply doc/source/getting-started/installation.rst]: No, I have deleted it.

**Comment by Edoardo-Pedicillo** [comment]
Thanks @MatteoRobbiati for your review, Could you explain how a description and a link are more readable than an image?
I agree that the images are not in harmony with the design of the web page, I suggest modifying instead of deleting them.

**Comment by MatteoRobbiati** [comment]
> Thanks @MatteoRobbiati for your review, Could you explain how a description and a link are more readable than an image? I agree that the images are not in harmony with the design of the web page, I suggest modifying instead of deleting them.

You are right on this.  I still think that the screenshot of all the page contains too much information: maybe we can cut it a bit? 
Sorry for the late answer.

**Comment by andrea-pasquale** [comment]
Thanks for all the updates @DavidSarlle.
To avoid triggering the pre-commit bot on every push you can install it locally using `pre-commit install`.
If you have any problems with that I can help.

**Comment by DavidSarlle** [comment]
@andrea-pasquale I'll take a look to the pre-commit. It was working before locally. Thanks. 

I have included all the docstrings for the calibration routines. 

Just a comment. The branch is pointing to "refactor-docs", that is quite away from the alvaro/fast_calibrate_qubit_states_pr, the most advanced branch (the latest branch containing many fixes and modifications in the calibration routines. Also includes 1 more characterization routine with docstrings). We will need to modify some docstrings accordingly with the changes in that branch.

**Review by andrea-pasquale — APPROVED** [review]
Thanks @Edoardo-Pedicillo  and @DavidSarlle.

**Pull request merged**